Wrapping/Generators/Python/itk/support/xarray.py

```diff
@@ -0,0 +1,139 @@
+# ==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ==========================================================================*/
+
+from typing import Tuple, TYPE_CHECKING
+
+import itk.support.types as itkt
+
+if TYPE_CHECKING:
+    try:
+        import xarray as xr
+    except ImportError:
+        pass
+
+
+__all__ = [
+    "xarray_from_image",
+    "image_from_xarray",
+]
+
+
+def xarray_from_image(l_image: "itkt.ImageOrImageSource") -> "xr.DataArray":
```

Comment:
Were `xarray_from_image` and `image_from_xarray` previously defined somewhere else? I see where they are defined here but not where they were removed from.

Reply:
Yes, they were in extras.py -- they have been removed from there.

Reply:
It looks like they are still present in this PR. Could you please take a look at `extras.py:627`?

Reply:
Ah, good catch! There was a copy in *Tests/extra.py*, which was removed, but not *extras.py* -- fixed now!

Reply:
😕 The copy I am seeing that still exists in the Github diff is at `itk/support/extras.py:627`. Please see my other comment below as well.

Reply:
I had some issues with the rebase -- just pushed an update.